src/Config/Auth.php

```diff
@@ -94,10 +94,12 @@ class Auth extends BaseConfig
      * to apply any logic you may need.
      */
     public array $redirects = [
-        'register'    => '/',
-        'login'       => '/',
-        'logout'      => 'login',
-        'force_reset' => '/',
+        'register'                => '/',
+        'login'                   => '/',
+        'logout'                  => 'login',
+        'force_reset'             => '/',
+        'after_permission_denied' => '/',
+        'after_group_denied'      => '/',
```

Comment:
It seems `after_` is not needed.
```suggestion
        'permission_denied' => '/',
        'group_denied'      => '/',
```

Reply:
Done.